benchmarks/optimize.py

```diff
@@ -0,0 +1,70 @@
+from typing import cast
```

Comment:
Do we need to include the `benchmarks` module into the `optuna` distribution package? If not, could you exclude the `benchmark` directory?

When I built a dist file with `python setup.py sdist` and installed the tar ball, I can import the `benchmarks`. This is due to the sdist file includes the `benchmarks` directory.